core/src/components/badge/badge.ionic.scss

```
@@ -1,5 +1,6 @@
 @use "../../themes/ionic/ionic.globals.scss" as globals;
 @use "./badge.common";
+@import "./badge.ionic.vars";
```

We should be using `@use` since `@import` is deprecated.

```suggestion
@use "./badge.ionic.vars";
```